src/sidebar/store/modules/drafts.js

```diff
@@ -1,10 +1,12 @@
 import { createSelector } from 'reselect';
 
-import * as metadata from '../../helpers/annotation-metadata';
 import { createStoreModule, makeAction } from '../create-store';
 import { removeAnnotations } from './annotations';
 
-/** @typedef {import('../../../types/api').Annotation} Annotation */
+/**
+ * @typedef {import('redux-thunk/extend-redux')} Dummy
```

Comment:
? educate me?

Reply:
This import, when put anywhere in a project, has the effect of modifying the type of the `dispatch` method of Redux stores to reflect the usage of `redux-thunk`. See notes at https://github.com/reduxjs/redux-thunk/blob/6f392f14787d81671b3ba3000244dbcbc0162628/extend-redux.d.ts#L13.